src/v2/Apps/Components/AppShell.tsx

```diff
@@ -18,6 +18,8 @@ import { AppToasts } from "./AppToasts"
 import { useNavBarHeight } from "v2/Components/NavBar/useNavBarHeight"
 import { useProductionEnvironmentWarning } from "v2/Utils/Hooks/useProductionEnvironmentWarning"
 import { useAuthValidation } from "v2/Utils/Hooks/useAuthValidation"
+import { IntlProvider } from "react-intl"
+import locale_en_us from "../../compiled-lang/en-us.json"
```

Comment:
Follow-up: I'm curious what including all of the different languages would amount to file-size-wise. In the future we should probably code-split via `import()` and `react-loadable` as we do with our other split components [as seen here](https://github.com/artsy/force/blob/main/src/v2/Apps/Artist/artistRoutes.tsx#L12-L17).